### PR TITLE
Feat/change opening order kzg

### PR DESCRIPTION
**Pull request by ThomasPiellard**
@Tabaie I just changed the order of the polynomials during the batch opening at zeta in plonk (see in prove.go where `BatchOpenSinglePoint` is called, same in the verifier where `BatchVerifyMultiPoints` is called). It's to be consistant with with the plonk verifier contract

### CODE REVIEW

File: backend/plonk/bls12-377/verify.go [hunk 1]
```diff
@@ -137,16 +137,16 @@ func Verify(proof *Proof, vk *VerifyingKey, publicWitness fr.Vector) error {
...
-	claimedQuotient := claimedValues[0]
-	linearizedPolynomialZeta := claimedValues[1]
-	l := claimedValues[2]
-	r := claimedValues[3]
-	o := claimedValues[4]
-	s1 := claimedValues[5]
-	s2 := claimedValues[6]
+	// qC := make([]fr.Element, len(proof.Bsb22Commitments))
+	// claimedValues := proof.BatchedProof.ClaimedValues[len(proof.Bsb22Commitments):]
+	// claimedValues := proof.BatchedProof.ClaimedValues[7:]
```
Comment (Tabaie) [inline backend/plonk/bls12-377/verify.go:142]: Wanna get rid of these?

File: backend/plonk/bn254/to_delete.go
Comment (Tabaie) [inline backend/plonk/bn254/to_delete.go:1]: This one's begging to be deleted 😁

**Review by Tabaie — CHANGES_REQUESTED** [review]
All good, except that a few lines and a file should be deleted.

**Comment by gbotrel** [comment]
@ThomasPiellard fix lines + merge? :)

**Review by Tabaie — APPROVED** [review]

**Pull request merged**